rewrite-java-test/src/test/java/org/openrewrite/java/RemoveImportTest.java

```diff
@@ -331,6 +331,86 @@ public class A {
         );
     }
 
+    @Test
+    void doNotLeaveLeaveInitialBlankLine() {
```

Comment:
Added this test case to assert a case (removing the first import, with no package statement) that was present in other test classes that uses RemoveImport.